app/electron/main/updater.js

```diff
@@ -1,5 +1,6 @@
 import {dialog} from 'electron';
-import {autoUpdater} from 'electron-updater';
+import pkg from 'electron-updater';
+const {autoUpdater} = pkg;
```

Comment:
This module is CJS, and this is the recommended workaround [according to the `electron-vite` docs](https://electron-vite.org/guide/dev#migrating-to-esm).